### PR TITLE
Improve long explanation for E0546

### PR DESCRIPTION
**Pull request by jesusprubio**
Helps with #61137

### CODE REVIEW

**Comment by jesusprubio** [comment]
@GuillaumeGomez I didn't add the reference to #61137 when I pushed, so I don't know if you're going to receive a notification.

File: compiler/rustc_error_codes/src/error_codes/E0546.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ fn unstable_fn() {}
...
 
-To fix the issue you need to provide a feature name.
+To fix the issue you need to provide the `feature` field.
 
 ```
 #![feature(staged_api)]
 #![stable(since = "1.0.0", feature = "test")]
 
-#[unstable(feature = "unstable_fn", issue = "none")] // ok!
+#[unstable(feature = "unstable_fn", issue = "none")]
```
Comment (GuillaumeGomez) [inline compiler/rustc_error_codes/src/error_codes/E0546.md:22]: Please keep the `// ok!`.

Comment (jesusprubio) [reply compiler/rustc_error_codes/src/error_codes/E0546.md:22]: Ok, I'm going to add it also here then. Isn't it? :)

https://github.com/rust-lang/rust/pull/81830

Comment (GuillaumeGomez) [reply compiler/rustc_error_codes/src/error_codes/E0546.md:22]: Arf indeed! Sorry, didn't pay enough attention...

Comment (jesusprubio) [reply compiler/rustc_error_codes/src/error_codes/E0546.md:22]: Don't worry, I think now I have everything to do it properly in the net PRs. Thanks!

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup squash

**Pull request merged**